docs/fundamentals/client/client.md

```diff
@@ -668,6 +663,20 @@ with Flow().add() as f, open('output.txt', 'w') as fp:
     )
 ```
 
+````{admonition} What errors can be handled by the callback
+:class: caution
+Callback can handle errors that are caused inside the Flow, it can be from an Executor raising an Exception or from the Gateway having connectivity errors with the Executors.
+It will not receive errors of connectivity between the Client and the Gateway.
+````
+
+### Transient fault handling with retries
+
+`client.post()` accepts `num_retries` and `retry_delay` parameters to control the capacity to retry requests when a transient connectivity error occurs.
```

Comment:
Can we expecitly explain what these two parameters control ?

Reply:
Okey, I will explain better